pgcli/pgcompleter.py

```diff
@@ -30,6 +31,7 @@ def __init__(self, smart_completion=True, pgspecial=None):
         super(PGCompleter, self).__init__()
         self.smart_completion = smart_completion
         self.pgspecial = pgspecial
+        self.keyword_counter = KeywordCounter(self.keywords)
```

Comment:
Instead of calling it `self.keyword_counter` let's call it `self.recommendation_engine` or something more generic. 

The idea is that in the future we might have different learning algorithms, one based on counting, one based on weighted frequency, or neural networks or something equally crazy etc. :)

We should also make `KeywordCounter()` take in a initial corpus of input strings train the model with the query history.